dataParser.py

```diff
@@ -55,4 +56,15 @@ def parse_time(self, time=""):
         values["start_time"] = datetime.time(int(times[0].split(":")[0]), int(times[0].split(":")[1]), 0, 0)
         values["end_time"] = datetime.time(int(times[1].split(":")[0]), int(times[1].split(":")[1]), 0, 0)
         
-        return values
\ No newline at end of file
+        return values
+
+    
+    def parse_additional_credit_data(self, section_information=""):
+        initial_index = section_information.find("Precludes additional credit for ")
+        if initial_index == -1:
+            return []
+        initial_index+=len("Precludes additional credit for ")
+        end_index = section_information.find(".", initial_index)
+        prereqs = section_information[initial_index:end_index]
+        prereqs = prereqs.split(", ")
```

Comment:
Add a line that removes "(this course is no longer offered)", I don't think we should include that.